tasks/lint-all.js

```diff
@@ -63,20 +64,7 @@ fluid.grunt.lintAll.defaults = {
     },
     markdownlint: {
         options: {
-            config: {
-                // See https://github.com/DavidAnson/markdownlint#rules--aliases for rule names and meanings.
-                "no-duplicate-header": false, // We use duplicate nested headers, as in section 1 and 2 both have the same sub-section name.
-                "no-trailing-punctuation": false,  // This catches headings that are questions, which seems wrong.
-                "header-style": { style: "atx" }, // Although we use a mix, in discussion with various team members, we agreed to try this for now.
-                "no-inline-html": false, // Obviously we need HTML
-                "line-length": {
-                    line_length: 120,
-                    tables:      false,
-                    code_blocks: false
-                },
-                "code-block-style": false, // Allow both fenced and indented styles
-                "ol-prefix": {style: "ordered"} // 1. 2. 3. etc
-            }
+            config: grunt.file.readJSON("./node_modules/markdownlint-config-fluid/.markdownlintrc.json")
```

Comment:
Can we make the location configurable?  I'm fine with the fluid config being the default, but it should be configurable.

Reply:
Here's an example of [how we've used options elsewhere in the package](https://github.com/fluid-project/fluid-grunt-lint-all/blob/main/tasks/lint-all.js#L130).  In the same file, you can see the default config.  If you can load the JSON there and add that to the defaults, then people can override config options in their Gruntfile without having to create their own settings file (we won't to support both ways of working).

Reply:
Sorry, I'm still not quite clear on what's expected— wouldn't overriding `lintAll.markdownlint.options.config` in a user's Gruntfile work without any additional changes here?

Reply:
In the 10 days since I left my last comment, I've made [a first pass at rewriting this package to avoid using Grunt altogether](https://github.com/fluid-project/fluid-lint-all/pull/2).  Given that we absolutely cannot use `grunt.file.readJSON in that package, I'll take care of adding comparable functionality to the replacement package.  I'd suggest closing this one out.